extra/exports

```diff
@@ -420,6 +420,7 @@ tnt_datetime_strftime
 tnt_datetime_to_string
```

Comment:
1. Thanks for the spot - corrected;
2. `adjust` is documented in the original RFC here https://hackmd.io/@Mons/S1Vfc_axK#%D0%9F%D0%B0%D1%80%D0%B0%D0%BC%D0%B5%D1%82%D1%80-adjust 
I've reviewed docbot request and changelog from commit 43e10ed34 (which was step.1 of datetime saga), and they were generic enough to cover all steps of datetime development, including current one;
3. `adjust` is implicitly month related for the final result. I do not believe we need to do anything special for interval only arithmetic - they will also obey the same month rules for result calculation. You could request changes in behaviour in original RFC, at the moment it behaves according to it;
4. Good question, agreed it was not documented well enough (neither in commit message nor in RFC).
`excess` mode came from `DT_EXCESS` modefrom `c-dt`, and behaved simply - if original day number, before month or year interval application, was larger than could fit to the month of a result, then excessive days overlap to the next month. i.e.
```
tarantool> T = date.new{year = 2004, month = 1, day = 31}
---
...
tarantool> I = date.interval.new{month = 1, adjust = 'excess'}
---
...
tarantool> T+I
---
- 2004-03-02T00:00:00Z
...

there were 29 days in February 2004, thus 31st became 2 after addition of a month in excess mode. 


Here is modified commit message, hope it makes thing clearer now:

```

 datetime, lua: interval arithmetics
 
 Add interval arithmetics as date:add{} and date:sub{}, or
 overloaded '+' and '-' operators (__add/_sub metamethods).
 There is no special processing for timezone specific
 daylight saving time changes, which impacts result of arithmetics.
 This will be completed after Olsen database support added.
 
 If one needs to control a way arithmetic proceeded for days
 of months then there is optional argument to `:add()` or `:sub()`,
 which defines how we round day in month after arithmetic
 operation:
 
 - 'last' - mode when day snaps to the end of month, if happens:
 
   -- 29.02.2004 -> 31.03.2004
   dt:add{month = 1, adjust = "last" }
 
 - 'none' only truncation toward the end of month performed
   (**default mode**):
 
   -- 29.02.* -> 29.03.*
   dt:add{month = 1, adjust = "none" }
 
 - 'excess' - overflow mode, without any snap or truncation to
   the end of month, straight addition of days in month, stopping
   over month boundaries if there is less number of days.
 
   dt = new{year = 2004, month = 1, day = 31}
   dt:add{month = 1, adjust = 'excess'} -> 02.03.2004
 
   -- i.e. there were 29 days in February 2004, 2 excessive days
      from original date 31 make result 2nd March 2004
 
 Intervals objects, created via `datetime.interval.new()` call
 and objects of the same attributes are equivalent for interval
 arithmetics and exchangeable. i.e.
 
    dt + date.interval.new{year = 1, month = 2}
 
 is the same as
    dt + {year = 1, month = 2}
 
 Follow-up to #5941
```